### PR TITLE
test!: removes e2e benchmark tests for blocks >8MB, uses main branch for others

### LINKED ISSUE #3865: Refocusing big block tests on 8MB block sizes and simplifying maintenance
# Problem

Initially, for the big block test, the plan was to run e2e benchmark tests for block sizes of 8, 32, and 64MB. However, the focus later shifted to 8MB blocks. While the current constants in celestia-app are compatible with 8MB blocks, modifying those constants was necessary to produce larger blocks, which led to this [PR](https://github.com/celestiaorg/celestia-app/pull/3261), based on which the [e2e big block tests](https://github.com/celestiaorg/celestia-app/blob/2c53ddbe0bf0d6aa0e944f34fbd5aae34581f742/test/e2e/benchmark/throughput.go#L35) are run. 
Since the current focus is on 8MB blocks, maintaining that branch is no longer necessary, so

**Acceptance Criteria**
- [ ]  I'd suggest closing it
- [ ]  The e2e benchmark tests will be updated to use the latest from the `main` branch. This simplifies maintenance and reduces the workload. 
- [ ] As a result, other benchmark tests for block sizes of 32 and 64MB will also be removed until a concrete plan is developed for those, and the relevant constants are introduced to celestia-app.

### PR DESCRIPTION
**Pull request by staheri14**
Closes #3865

### CODE REVIEW

**Comment by staheri14** [comment]
The benchmark tests `TwoNodeBigBlock8MBLatency` and `TwoNodeBigBlock8MB`, using the latest `main` branch for the validators and txsim (i.e., using the changes in this PR), passed with no issues.

**Review by rootulp — DISMISSED** [review]

**Review by ninabarbakadze — DISMISSED** [review]

File: test/e2e/benchmark/throughput.go [hunk 1]
```diff
@@ -24,17 +24,17 @@ func getBigBlockManifest() (Manifest, error) {
...
+		MemoryLimit:   resource.MustParse("12Gi"),
+		CPU:           resource.MustParse("8"),
+		Volume:        resource.MustParse("20Gi"),
+	    },
+	    TxClientsResource: testnet.Resources{
+		MemoryRequest: resource.MustParse("1Gi"),
+		MemoryLimit:   resource.MustParse("3Gi"),
+		CPU:           resource.MustParse("2"),
+		Volume:        resource.MustParse("1Gi"),
+        },
```
Comment (rootulp) [inline test/e2e/benchmark/throughput.go:37]: the indentation looks off and golangci-lint is complaining: https://github.com/celestiaorg/celestia-app/actions/runs/10950431497/job/30405641643?pr=3866#step:5:32

Comment (ninabarbakadze) [reply test/e2e/benchmark/throughput.go:37]: once you fix lint I'll reapprove @staheri14

File: test/e2e/benchmark/throughput.go [hunk 2]
```diff
@@ -138,54 +143,23 @@ func runBenchmarkTest(logger *log.Logger, testName string, manifest Manifest) er
...
-	manifest.MaxBlockBytes = 8 * testnet.MB
+	manifest, err := getBigBlockManifest()
+	testnet.NoError("failed to get manifest", err)
 	manifest.Validators = 50
 	manifest.TxClients = 50
 	manifest.BlobSequences = 2
 	return runBenchmarkTest(logger, "LargeNetworkBigBlock8MB", manifest)
 }
-
-func LargeNetworkBigBlock32MB(logger *log.Logger) error {
```
Comment (rootulp) [inline test/e2e/benchmark/throughput.go:175]: I don't think this was part of the exported Go API https://pkg.go.dev/github.com/celestiaorg/celestia-app/v2@v2.2.0/test/e2e/benchmark so we can probably drop the `!` from the PR title

**Review by evan-forbes — COMMENTED** [review]
i'm good with removing these

in theory, we'll get a pulumi backend for knuu, which would allow us to use these, but we can add them back then

**Comment by staheri14** [comment]
With this proposal https://github.com/celestiaorg/celestia-app/issues/3865#issuecomment-2529014301 I think this PR is no longer needed.

**Pull request closed without merging**